src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -428,13 +452,57 @@ class SourceBox extends HTMLElement {
     }
   }
 
-  toggleShowHide() {
-    const element = this.querySelector("#open-close-icon");
-    element.classList.toggle("item-closed");
+  toggleShowHide(altKey) {
+    const cardElements = !altKey
+      ? [this]
+      : document.querySelectorAll(".fontra-ui-font-info-sources-panel-source-box");
+
+    const thisIconElement = this.querySelector("#open-close-icon");
+    const isClosed = thisIconElement.classList.contains("item-closed");
+
+    for (const cardElement of cardElements) {
+      const elementIcon = cardElement.querySelector("#open-close-icon");
+      if (isClosed) {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = false;
+        cardElement.classList.remove("item-closed");
+        elementIcon.classList.remove("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.remove("min-height");
+        }
+      } else {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = true;
+        cardElement.classList.add("item-closed");
+        elementIcon.classList.add("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.add("min-height");
+        }
+      }
+    }
+  }
+
+  getOneliner() {
+    const onelinerElement = html.div({
+      class: "fontra-ui-font-info-sources-panel-oneliner",
+    });
+    onelinerElement.appendChild(html.span({ class: "bold" }, [this.source.name] + " "));
+
+    for (const axis of this.fontAxesSourceSpace) {
+      const axisElement = document.createElement("span");
+      const sourceLocationValue = round(this.source.location[axis.name], 2);
```

Comment:
I'm not sure of soure.location is guaranteed to be a "dense" location: are we sure default values can't be omitted? If so, you need to fall back to `axis.defaultValue` if `axis.name` does not exist in the location. So probably we need to do this (untested):
```suggestion
      const sourceLocationValue = round(this.source.location.hasOwnProperty(axis.name) ? this.source.location[axis.name] : axis.defaultValue, 2);
```

Reply:
About _are we sure default values can't be omitted?_. 
You wrote _ensure it includes axes that are at the default_ here: https://github.com/googlefonts/fontra/issues/1466#issuecomment-2421786696
Did I misunderstood you?

Reply:
We could ignore the default axes and use locationToString for the oneliner axes info. This would make the whole code much simpler. Please let me know how to move forward.

Reply:
> Did I misunderstood you?

I think so. The formatting code is good: it adds axes that are at the default.

BUT: any location object _may leave out axis values to indicate they are at the default_. For example, the location `{}` means: all axes (if any) are at the default. But your code does not take this into account: `this.source.location[axis.name]` just assumes `axis.name` exists in the location object.

A way to test this is to add a new axis, then look at the sources panel. I bet there will be problems now or info missing.